cobbler.spec

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) 2006 Michael DeHaan <mdehaan@redhat.com>
```

Comment:
Is there a specific reason you added that line?

Reply:
> Is there a specific reason you added that line?

Please do not squash the patches, then the reason is in a separate submit message:
In openSUSE:Factory there are some additional tests, beside rpmlint. I'll ask whether they can be integrated there, as late error messages are counterproductive. Anyway the bot complains about:
Output of check script:
cobbler.spec does not appear to contain a Copyright comment. Please stick to the format
  
  # Copyright (c) 2011 Stephan Kulow

As Michael DeHaan has submitted the first patch for the spec file and for quite a while was the only submitter, it's certainly him, who should be listed there.